kythe/rust/fuchsia_extractor/src/bin/main.rs

```diff
@@ -875,16 +875,67 @@ mod testing {
         let ru_vname = ri_one_file.get_v_name();
         assert_eq!("fuchsia", ru_vname.get_corpus());
         assert_eq!(SOURCE_CORPUS_NAME, ru_vname.get_root());
-
-        let ru_info = ri_one_file.get_info();
+        // In vname, the "path" is relative to corpus root.
         assert_eq!(
             "third_party/rust_crates/vendor/nom/lib.rs",
-            ru_info.get_path(),
+            ru_vname.get_path(),
             concat!(
-                "Expected to be relative to the root of its respective corpus",
-                " for example for ../../dir/file.txt it will be dir/file.txt"
+                "Expected to be relative to the root dir of its ",
+                "respective corpus, for example ../../dir/file.txt ",
+                "should appear here as dir/file.txt"
             )
         );
+
+        let ru_info = ri_one_file.get_info();
+        // In FileInfo, the path is relative to the build directory.
+        assert_eq!(
+            "../../third_party/rust_crates/vendor/nom/lib.rs",
+            ru_info.get_path(),
+            "Expected to be relative to the build directory"
+        );
         assert_eq!("nom", cu_vname.get_signature());
     }
+
+    #[test]
+    #[ignore]  // Reenable once we can subprocess `kzip`.
```

Comment:
What errors do you see trying to run the binary?

Reply:
See here: https://groups.google.com/g/bazel-discuss/c/S-uSkgNcmhg
I suppose this is something about executable bits in the bazel sandbox, but I admit I've never seen this before.  The kzip binary is present in the sandbox, and I can execute it locally.

Reply:
Figured it out -- my binary `kzip` was `.../kythe/go/platform/tools/kzip/kzip`, not `.../kythe/go/platform/tools/kzip`.

With this fix, things work again.

Reply:
Ah, yeah, the Bazel go rules result in a lot of stuttering.

Reply:
As to how I discovered this, I tried to wrap my command line `FOO` into `sh -c $FOO`, at which point bash complained `can not execute a directory` or some such.  Which was very helpful.